distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -135,6 +136,10 @@
 
     private static final String DEFAULT_URL_RESOURCE_READER_ROOT_RESOURCE_DIRS = "data/products";
 
+    private static KarafConsole console;
+
+    private static final String CLEAR_CACHE = "catalog:removeall -f --cache";
```

Comment:
I wonder if we should use `-f -p` here  (using provider should wipe metacards of any tag instead of just resource tags)